tests/model_serving/model_runtime/mlserver/basic_model_deployment/test_sklearn_basic_model_deployment.py

```
@@ -0,0 +1,98 @@
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.pod import Pod
+from utilities.constants import Protocols
+from tests.model_serving.model_runtime.mlserver.constant import (
+    SKLEARN_REST_INPUT_QUERY,
+    SKLEARN_GRPC_INPUT_QUERY,
+    MODEL_PATH_PREFIX,
+)
+from tests.model_serving.model_runtime.mlserver.constant import (
+    BASE_RAW_DEPLOYMENT_CONFIG,
+    BASE_SEVERRLESS_DEPLOYMENT_CONFIG,
+)
```

Pylint (3.3.7)

[error] Unable to import 'tests.model_serving.model_runtime.mlserver.constant'

(E0401)